tests/core/zstream.c

```diff
@@ -68,7 +68,7 @@ void test_core_zstream__fails_on_trailing_garbage(void)
 
 	/* append some garbage */
 	for (i = 0; i < 10; i++) {
-		git_buf_putc(&deflated, i);
+		git_buf_putc(&deflated, (char)i);
```

Comment:
Nit: you could also just make `i` a `char`. Not that it'd matter much

Reply:
Yeah, idk why I didn't do that.  This looks dumb.

Reply:
Fixed.